internal/compiler-interface/src/main/java/xsbti/compile/IncrementalCompilerUtil.java

```diff
@@ -0,0 +1,63 @@
+package xsbti.compile;
+
+import java.io.File;
+
+public interface IncrementalCompilerUtil {
+    /**
+     * Return the default instance of {@link }
+     * @return
+     */
+    IncrementalCompiler defaultIncrementalCompiler();
+
+
+    /**
+     * Create a Scala compiler from a {@link ScalaInstance}, the jar containing
+     * the compiler interface to be used and {@link ClasspathOptions}.
+     *
+     * @param scalaInstance The Scala instance to be used.
+     * @param compilerBridgeJar The jar file of the compiler bridge.
+     * @param classpathOptions The options of all the classpath that the
+     *                         compiler takes in.
+     *
+     * @return A Scala compiler with the given configuration.
+     */
+    ScalaCompiler scalaCompiler(ScalaInstance scalaInstance,
```

Comment:
Just so it's clear and we agree to what we're doing:

this is the API that @eed3si9n had intentionally removed from public API in #30, now brought back.